### PR TITLE
[FEATURE] Auth Management #33

### PR DESCRIPTION
**Pull request by theCodeCrusaderX**
all the functionality that you wanted is done .

one more thing that u wanted session based auth so its being new to me but somehow due to doc i manage to do with session based auth thats why it took little bit more time if i will be doing as jwt and it would be easier for me but i challenge myself and did this 

image

one more thing i added a extra route for logout i know i can be done without doing this but belive me its good practice deling logout feature using saparate route . and also its being easy for me to clear the session after the user logout.

thats all thank u for giving me this opportunity.

### CODE REVIEW

**Comment by pooranjoyb** [comment]
Please resolve the conflicts.

**Comment by theCodeCrusaderX** [comment]
now check

**Comment by theCodeCrusaderX** [comment]
ahh.. all conflicts are resolved now you can merge it

**Comment by theCodeCrusaderX** [comment]
is everything ok ? when you will merge :)

**Comment by pooranjoyb** [comment]
> is everything ok ? when you will merge :)

Some minor conflicts, fix them, else all fine. will merge

**Comment by theCodeCrusaderX** [comment]
done

File: views/pages/profile.ejs [hunk 1]
```diff
@@ -258,7 +224,118 @@
 
   </body>
 
-  <script defer src="/js/auth.js"></script>
+  <!-- handeling form data -->
```
Comment (pooranjoyb) [inline views/pages/profile.ejs:227]: You removed the Auth.js file. please check that script should be kept in /js/ folder in public dir

**Review by pooranjoyb — CHANGES_REQUESTED** [review]
also i cannot see the user data in the Cookies after the Successful login :/

**Pull request closed without merging**